docs/src/pages/components/typography/Types.tsx

```diff
@@ -1,19 +1,15 @@
 import * as React from 'react';
+import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
-import { makeStyles } from '@material-ui/core/styles';
-
-const useStyles = makeStyles({
-  root: {
-    width: '100%',
-    maxWidth: 500,
-  },
-});
 
 export default function Types() {
-  const classes = useStyles();
-
   return (
-    <div className={classes.root}>
+    <Box
+      sx={{
+        width: '100%',
+        maxWidth: 500,
+      }}
+    >
```

Comment:
We aim to maximize density 

```suggestion
    <Box sx={{ width: '100%', maxWidth: 500 }}>
```

Reply:
Resolved